### PR TITLE
Support for milliseconds intervals

### PR DESCRIPTION
**Pull request by cuchi**
With this, the interval can be defined using a more flexible manner than **cron**, which was discussed in #942

For example, when I need a job to execute every 90 minutes, I would do like that:
```javascript
queue.add({ ... }, { repeat: { seconds: 5400 } })
```

I am not sure if it is the best API design for this, maybe the contributors have a better opinion on that. I need to update the reference and the TS definitions too.

We at @rung-tools will be testing this branch with our application to see if it is stable enough, I'll report here if something is not right.

### CODE REVIEW

File: lib/repeatable.js [hunk 1]
```diff
@@ -127,13 +127,23 @@ module.exports = function(Queue) {
...
+      : String(repeat.milliseconds || repeat.seconds * 1000);
+
+    return name + ':' + jobId + endDate + suffix;
   }
 
-  function getNextMillis(millis, cron, opts) {
+  function getNextMillis(millis, opts) {
+    var millisInterval = opts.milliseconds || opts.seconds * 1000;
+    if (millisInterval) {
+      return new Date(millis + millisInterval);
```
Comment (manast) [inline lib/repeatable.js]: Can't we just return ```millis+millisInterval``` ?

File: lib/repeatable.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ module.exports = function(Queue) {
     var now = Date.now();
     now = prevMillis < now ? now : prevMillis;
 
-    var nextMillis = getNextMillis(now, repeat.cron, repeat);
+    var nextMillis = getNextMillis(now, repeat);
```
Comment (manast) [inline lib/repeatable.js]: maybe an exception should be thrown if the user uses both the milliseconds and the cron expression at the same time.

**Comment by manast** [comment]
Great job!, thanks for the contribution. Regarding the api, another possibility could be to have an option called ```every``` and then only support milliseconds, or allowing a postfix s for seconds, ms for ms, etc.

**Comment by manast** [comment]
@cuchi bump!

File: test/test_repeat.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ var ONE_HOUR = 60 * ONE_MINUTE;
 var ONE_DAY = 24 * ONE_HOUR;
 var MAX_INT = 2147483647;
 
-describe('repeat', function() {
+describe.only('repeat', function() {
```
Comment (manast) [inline test/test_repeat.js]: remove .only

Comment (cuchi) [reply test/test_repeat.js]: I did it as soon as I realized in [8ae7a58](https://github.com/OptimalBits/bull/pull/944/commits/8ae7a589fff078e8f77c3e06e378cca21a5e9444).

**Pull request merged**